src/main/java/org/dependencytrack/policy/cel/CelPolicyScriptHost.java

```diff
@@ -169,10 +172,13 @@ private static MultiValuedMap<Type, String> analyzeRequirements(final CheckedExp
                         requirements.put(TYPE_COMPONENT, "version");
                     }
                 }
+                case FUNC_COMPARE_VERSION_DISTANCE -> {
+                    requirements.putAll(TYPE_COMPONENT, List.of("purl", "uuid", "version"));
+                    requirements.put(TYPE_REPOSITORY_META_COMPONENT, "latestVersion");
```

Comment:
The latest version is a field on the `Component` Protobuf type, and is named `latest_version`.

```suggestion
                    requirements.put(TYPE_COMPONENT, "latest_version");
```